certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -479,6 +479,127 @@
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "accountId",
+            "in": "path",
+            "description": "Account id",
+            "required": true,
+            "x-ms-summary": "Account",
+            "x-ms-test-value": "insert account id",
+            "x-ms-dynamic-values": {
+              "operationId": "GetLoginAccounts",
+              "value-collection": "loginAccounts",
+              "value-path": "accountIdGuid",
+              "value-title": "name"
+            },
+            "type": "string"
+          },
+          {
+            "name": "envelopeId",
+            "in": "path",
+            "description": "Envelope id",
+            "required": true,
+            "x-ms-summary": "Envelope",
+            "x-ms-test-value": "insert envelope id",
+            "type": "string"
+          },
+          {
+            "name": "signerName",
+            "in": "query",
+            "description": "Name of The Signer",
+            "required": true,
+            "x-ms-summary": "Signer Name",
+            "x-ms-test-value": "test name",
+            "x-ms-visibility": "important",
+            "type": "string"
+          },
+          {
+            "name": "signerEmail",
+            "in": "query",
+            "description": "Email of The Signer",
+            "required": true,
+            "x-ms-summary": "Signer Email",
+            "x-ms-test-value": "test@email.com",
+            "x-ms-visibility": "important",
+            "type": "string"
+          },
+          {
+            "name": "clientUserId",
+            "in": "query",
+            "description": "Client User ID",
+            "required": true,
+            "x-ms-summary": "Client User ID",
+            "x-ms-visibility": "important",
+            "type": "string"
+          },
+          {
+            "name": "authenticationMethod",
+            "in":"query",
+            "description": "Authentication Method",
+            "required":true,
+            "x-ms-summary": "Authentication Method",
+            "x-ms-visibility": "important",
+            "type":"string",
+            "enum":["Biometric", "Email", "HTTPBasicAuth", "Kerberos", "KnowledgeBasedAuth", "None", "PaperDocuments", "Password", "RSASecureID", "SingleSignOn_CASiteminder", "SingleSignOn_InfoCard", "SingleSignOn_MicrosoftActiveDirectory", "SingleSignOn_Other", "SingleSignOn_Passport", "SingleSignOn_SAML", "Smartcard", "SSLMutualAuth", "X509Certificate"],
+            "default": "None"
+          },
+          {
+            "name": "returnUrl",
+            "in":"query",
+            "description": "Return URL",
+            "required":true,
+            "x-ms-summary": "Return URL",
+            "x-ms-visibility":"important",
+            "type":"string",
+            "enum":["Default URL", "Add A Different URL"],
```

Comment:
lower case the D in Different

Reply:
Add a different Url?

Reply:
Yes. The return URL is the redirect URL. We allow the sender to either use our redirect URL or they can use their redirect URL by adding a different one.